pom.xml

```diff
@@ -1406,6 +1382,59 @@
           <artifactId>maven-deploy-plugin</artifactId>
           <version>2.8.2</version>
         </plugin>
+
+	<!-- This plugin's configuration is used to store Eclipse m2e settings only. -->
+	<!-- It has no influence on the Maven build itself. -->
+	<plugin>
+	  <groupId>org.eclipse.m2e</groupId>
```

Comment:
It is ide specific. Eclipse / scala-ide show an error by default when it see a plugin which it does not understand. Adding this to a pom file is, imho, better than adding a readme of how to setup an ide.

In this case it is saying "hey ide, don't worry about the custom configuration of maven-jar-plugin" because nobody use ide to generate the jar anyway.